charts/simpledataexchanger/values.yaml

```diff
@@ -97,17 +97,17 @@ backend:
   #    hosts:
   #      - chart-example.local
 
-  resources: {}
+  resources:
   # We usually recommend not to specify default resources and to leave this as a conscious
   # choice for the user. This also increases chances charts run on environments with little
   # resources, such as Minikube. If you do want to specify resources, uncomment the following
   # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  # limits:
-  #   cpu: 100m
-  #   memory: 128Mi
-  # requests:
-  #   cpu: 100m
-  #   memory: 128Mi
+    limits:
+      cpu: 800m
+      memory: 2Gi
```

Comment:
Please set memory request and limit equal according to this [TRG](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-04#memory)